docs/conf.py

```diff
@@ -233,8 +236,32 @@ def copy_legacy_redirects(app, docname): # Sphinx expects two arguments
             if os.path.isfile(src_path):
                 copyfile(src_path, target_path)
 
-# -- Finally, Setup -------------------------------------------------------
+def translate_schema_and_codelists(language='en'):
```

Comment:
Should this function actually be called from somewhere, or does that come later?

Reply:
That's coming soooooon!